midend/parserUnroll.cpp

```diff
@@ -386,7 +417,7 @@ class ParserSymbolicInterpreter {
                                              const IR::Expression*>;
 
     EvaluationSelectResult evaluateSelect(ParserStateInfo* state,
-                                          ValueMap* valueMap) {
+                                   ValueMap* valueMap) {
```

Comment:
why change the indentation here?

Reply:
I don't even know, it happened automatically, I'll fix it